src/main/java/org/pentaho/mongo/wrapper/ConnectionStringMongoClientWrapper.java

```diff
@@ -0,0 +1,380 @@
+/*!
+ * Copyright 2010 - 2021 Hitachi Vantara.  All rights reserved.
```

Comment:
If this file is new as of this year, the only date it needs is 2021.  The dates on each file should reflect how long the file has been in the baseline.